draft-irtf-cfrg-frost.md

```diff
@@ -842,8 +842,9 @@ and publishes the resulting signature.
   Inputs:
   - group_commitment, the group commitment returned by compute_group_commitment,
     an Element in G.
-  - sig_shares, a set of signature shares z_i, Scalar values, for each signer,
-    of length NUM_SIGNERS, where MIN_SIGNERS <= NUM_SIGNERS <= MAX_SIGNERS.
+  - sig_shares, a set of signature shares z_i, Scalar values, for each signer
+    that participated in Rounds 1 and 2, of length NUM_SIGNERS, where
```

Comment:
Rather than note this in the parameter list, I would say this in lines L838-839. Maybe something like:

```
Otherwise, if all shares from signers that participated in Rounds 1 and 2 are valid, the 
Coordinator performs the `aggregate` operation and publishes the resulting signature. 
```

Reply:
This looks like a good change to me

Reply:
Done in 4becfcf